### ISSUE TITLE
Neighbourhood filter concept map

### ISSUE THREAD

**Issue by manerotoni** [NEUBIAS/training-resources#320]
The concept map looks that it is applying the mathematical formula to the pixel coordinate rather than pixel values.

Modify it to make it more clear.

**Comment by tischi** [comment]
I don't think so. It says "Pixel neighbourhood" and we define in our teaching of digital images that  Pixel = Value & Coordinate. 
So I think this is perfect like it is...
